src/modm/board/disco_f469ni/module.lb

```diff
@@ -14,7 +14,7 @@
 def init(module):
     module.name = ":board:disco-f469ni"
     module.description = FileReader("module.md")
-    # Revisions = [b-03, b-02, b-01]
+    # Revisions = [c-01, b-08, b-03, b-02, b-01]
```

Comment:
If I remenber correctly the first revision in the list is the "default" revision. Not sure if we should change the default revision to ´C-01´ (yet)?

Reply:
Maybe not (yet)?
Also - do you happen to have one of the `b-08` or `c-01` revision boards, too?

Reply:
> do you happen to have one of the `b-08` or `c-01` revision boards, too?

Not yet

Reply:
Ok :-D I just ordered a second one - just to make sure, that my first display isn't defective :-D